### PR TITLE
VB-325: fix category property

### PR DESCRIPTION
**Pull request by voonya**

### CODE REVIEW

**Review by YevheniiBieliaiev — APPROVED** [review]

File: packages/frontend/components/save-post/description.tsx [hunk 1]
```diff
@@ -20,6 +22,8 @@ export default function DescriptionBlock({
 
   const { t } = useTranslation();
 
+  ConditionFields(t);
```
Comment (Nightlngale) [inline packages/frontend/components/save-post/description.tsx:25]: why?

File: packages/frontend/components/save-post/form-utils.ts [hunk 2]
```diff
@@ -48,3 +49,14 @@ export const SellerFields = (t: TFunction): SellerFieldsType => ({
     title: t('create-post:sellerSelect.otherUser'),
   },
 });
+
+export const ConditionFields = (t: TFunction): ConditionFieldsType => ({
+  NEW: {
```
Comment (Nightlngale) [inline packages/frontend/components/save-post/form-utils.ts:54]: It's better to use [ConditionFieldsType.new]

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/save-post/form-utils.ts (hunk 2 above)
Comment (voonya) [reply packages/frontend/components/save-post/form-utils.ts:54]: image

Comment (Nightlngale) [reply packages/frontend/components/save-post/form-utils.ts:54]: yes, because it's not enum, but why? use enum or at least camelCase for properties

**Review by Nightlngale — CHANGES_REQUESTED** [review]

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**